admin/openapi-1.2.39.yaml

```diff
@@ -7643,6 +7791,102 @@ components:
         - status
         - lpType
       description: ''
+    Smartlink:
+      title: Smartlink
+      type: object
+      x-tags:
+        - Smartlink
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        smartLinkEvent:
```

Comment:
smartlink.smartLinkEvent ?? redundant naming convention, plz update it to just use "event" and do the changes in the API as well